vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/impl/JcrPackageRegistry.java

```diff
@@ -115,6 +115,11 @@
      * the package root prefix of the primary root path.
      */
     private final String primaryPackRootPathPrefix;
+    
+    /**
+     * FSPackageRegistry can be registered if present in the system to be able to look up presatisfied dependencies
+     */
+    private FSPackageRegistry fsPackageRegistry = null;
```

Comment:
does this need to be a `FSPackageRegistry` or could this just be a _base_ or _fallback_ package registry?

```suggestion
    private PackageRegistry baseRegistry = null;
```